sos/report/plugins/kubernetes.py

```diff
@@ -105,12 +134,16 @@ def setup(self):
                     continue
                 node = words[0]
                 self.add_cmd_output(
-                    "%s describe node %s" % (self.kube_cmd, node),
+                    f"{self.kube_cmd} describe node {node}",
                     subdir='nodes'
                 )
+        if self.get_option('all'):
```

Comment:
same here, I would not gate this behind all.

Reply:
This is changed too.